Parking/short_term_parking.csv

```diff
@@ -0,0 +1,3 @@
+name,description,note,latitude,longitude
+Ira G. Needles Hall (NH),,,43.46954,-80.544

```

Comment:
Nope, the data came from: https://uwaterloo.ca/map/data/short-term_parking.txt .
But, I did notice that I dropped some of the data that would be nice to have for short term parking. I'll update this file with the missing information.